docs/content/integrations/dbt/using-dbt-with-dagster/load-dbt-models.mdx

```diff
@@ -115,49 +109,42 @@ The most important file is the Python file that contains the set of definitions
 
 In our `definitions.py` Python file, we import from `assets.py`, which contains the code to model our dbt models as Dagster assets. To return a Dagster asset for each dbt model, the code in this `assets.py` file needs to know what dbt models you have. It finds out what models you have by reading a file called a `manifest.json`, which is a file that dbt can generate for any dbt project and contains information about every model, seed, snapshot, test, etc. in the project.
 
-To retrieve the `manifest.json`, `assets.py` imports from `constants.py`, which defines the path to the `manifest.json` file:
+To retrieve the `manifest.json`, `assets.py` imports from `project.py`, which defines an internal representation of your dbt project. Then, in `assets.py`, the path to the `manifest.json` file can be accessed with `dbt_project.manifest_path`:
 
-```python file=integrations/dbt/tutorial/load_dbt_models/constants.py startafter=start_load_manifest endbefore=end_load_manifest
-import os
+```python file=integrations/dbt/tutorial/load_dbt_models/project.py startafter=start_load_project endbefore=end_load_project
 from pathlib import Path
 
-from dagster_dbt import DbtCliResource
+from dagster_dbt import DbtProject
 
-dbt_project_dir = Path(__file__).joinpath("..", "..", "..").resolve()
-dbt = DbtCliResource(project_dir=os.fspath(dbt_project_dir))
-
-# If DAGSTER_DBT_PARSE_PROJECT_ON_LOAD is set, a manifest will be created at run time.
+dbt_project = DbtProject(
+    project_dir=Path(__file__).joinpath("..", "..", "..").resolve(),
+)
+# If `dagster dev` is used, the dbt project will be prepared at run time, and manifest will be created.
```

Comment:
```suggestion
# If `dagster dev` is used, the dbt project will be prepared to create the manifest at run time.
```

Reply:
All changes made in [633ba87](https://github.com/dagster-io/dagster/pull/23042/commits/633ba87b5024b8a12fda6df9cc1ff20a395183fc)